neard/Cargo.toml

```
@@ -25,7 +25,7 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 shell-escape = "0.1.5"
 thiserror = "1"
-tikv-jemallocator = { version = "0.4.0", optional = true }
+tikv-jemallocator = { version = "0.4.0" }
```

```suggestion
tikv-jemallocator = "0.4.0"
```

I've also noticed that there's 0.5 available, should update to that at some point (which I belive includes updating rocksdb to use that as well)

---

I noticed that too.  I figured best to handle that in a separate PR.